src/app/lib/utilities/getPassportHome/index.js

```diff
@@ -1,18 +1,36 @@
 import pathOr from 'ramda/src/pathOr';
 
-const getPassportHome = pageData => {
+export const getPassportHome = pageData => {
   const homeServiceUrl = pathOr(
     null,
     ['metadata', 'passport', 'home'],
     pageData,
   );
-  return pageData && homeServiceUrl
-    ? homeServiceUrl
-        .split('/')
-        .slice(-1)
-        .pop()
-        .toLowerCase()
-    : null;
+  const passportHomeOverride = pathOr(
+    null,
+    ['metadata', 'passportHome'],
+    pageData,
+  );
+  return (
+    passportHomeOverride ||
+    (pageData && homeServiceUrl
+      ? homeServiceUrl
+          .split('/')
+          .slice(-1)
+          .pop()
+          .toLowerCase()
+      : null)
+  );
+};
+
+export const isValidPassportHome = (passportHome, service) => {
+  const overrides = {
+    brasil: 'portuguese',
```

Comment:
I think its safer to flip this mapping, asserting a service can have a home vs home can be on a service

Reply:
Also I _think_ this means the `portuguese` service can show content with a home of `portuguese` OR `brasil`, not just only `brasil`. Not sure if this matters, worth to check with @jimjohnsonrollings

Reply:
I think we should only be allowing the one we think is correct - no guaruntee someone won't create a passport with the other name in the future

Reply:
I think something like
```js
export const isValidPassportHome = (passportHome, service) => {
  if (!passportHome) return true;

  const overrides = {
    portuguese: 'brasil',
  };

  if (overrides[service]) {
    return overrides[service].includes(passportHome)
  }

  return passportHome === service
};
```
Might work :)

Reply:
Is it impossible that `Portuguese` can map to a passport home besides `Brasil`?

Reply:
I'm fairly sure it is impossible

Reply:
Fixed!